### ISSUE TITLE
[wifid] iw command is missing (v0.3.5)

### ISSUE THREAD

**Issue by cocochpie** [spr-networks/super#286]
When I updated v0.3.5, the spr stopped working, so I proceeded with the full reset. (Both wired and wireless went wrong)

After that, I kept seeing wifid-related error messages on the admin screen, so when I checked the container log, there was a message as below. 

maybe the 'iw' package is missing?

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
/scripts/startup.sh: line 33: iw: command not found
wlan0: interface state UNINITIALIZED->COUNTRY_UPDATE
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     6  100     6    0     0      3      0  0:00:02  0:00:01  0:00:01     3
wlan1: interface state UNINITIALIZED->COUNTRY_UPDATE
/scripts/startup.sh: line 33: iw: command not found
```

**Comment by cocochpie** [comment]
quick remedy:

docker compose exec -it wifid
apt update && apt install -y iw

then goto admin wifi menu -> "restart all wifi devices"

**Comment by cocochpie** [comment]
nice, thank you always for the super great spr.

**Comment by lts-rad** [comment]
fixed in v0.3.6
